.github/workflows/master-to-main.yml

```diff
@@ -0,0 +1,77 @@
+name: Convert the master branch to the main branch and set up a monorepo
+
+on:
+  workflow_dispatch:
+
+jobs:
+  master-to-main:
+    if: github.triggering_actor == "nickvigilante"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.5.0
+        with:
+          ref: master-2
+          path: source
+      - uses: actions/checkout@v3.5.0
+        with:
+          ref: main-2
+          path: target
+      - name: Copy new file changes from master to main
+        run: |
+          cp -f source/v1.0 target/docs/docs
```

Comment:
Are you sure you wanted double nested `docs/docs`? Looks like this: https://github.com/cockroachdb/docs/tree/main-2/docs/docs

And shows up like this:

![Screenshot 2023-04-04 at 10 04 28 AM](https://user-images.githubusercontent.com/101147019/229818161-6c1e3ea9-ffb1-45f9-8c42-05591978efec.png)

Reply:
Yes, because I ideally will have something like this:

```
.
├── CODE_OF_CONDUCT.md
├── CONTRIBUTING.md
├── LICENSE
├── README.md
├── StyleGuide.md
├── docs
│   ├── advisories
│   ├── api
│   ├── cockroachcloud
│   ├── releases
│   ├── tutorials
│   ├── v1.0
│   ├── v1.1
│   ├── v19.1
│   ├── v19.2
│   ├── v2.0
│   ├── v2.1
│   ├── v20.1
│   ├── v20.2
│   ├── v21.1
│   ├── v21.2
│   ├── v22.1
│   ├── v22.2
│   └── v23.1
├── githooks
│   └── pre-commit
├── navgoco_license
├── netlify.toml
├── package.json
└── vale
    ├── CockroachDB
    │   ├── Abbreviations.yml
    │   ├── Contractions.yml
    │   ├── HeadingsCase.yml
    │   ├── HeadingsPunctuation.yml
    │   ├── Hyperbolic.yml
    │   ├── Inclusivity.yml
    │   ├── NondescriptLinkTitles.yml
    │   ├── OrderedLists.yml
    │   ├── OxfordComma.yml
    │   └── Terminology.yml
    ├── Vocab
    │   └── CockroachDB
    └── vale
        ├── GenderBias.yml
        ├── Hedging.yml
        ├── Redundancy.yml
        ├── Repetition.yml
        └── Uncomparables.yml
```

And each version will be pulled out from `docs/docs` to `docs/tutorials` for example. Just thinking ahead.

Reply:
And each of those `docs/X` folders will be a fully self-contained Jekyll site that can be built on its own without the need to build everything.

Reply:
Awesome, smart thinking! Thanks for the explanation!